docs/guides/networking/linode-network/ip-failover-bgp-frr/index.md

```diff
@@ -0,0 +1,300 @@
+---
+slug: ip-failover-bgp-frr
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: "Learn how to use Linode's IP Sharing feature to configure IP failover using FRR, a routing software that implements BGP"
+keywords: ['IP failover','elastic IP','frr','bgp']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2022-01-11
+modified_by:
+  name: Linode
+title: "Configuring IP Failover using FRR (BGP)"
+contributor:
+  name: Linode
+external_resources:
+- '[FRRouting Documentation](http://docs.frrouting.org/en/latest/overview.html)'
+---
+
+{{<note>}}
+Not all data centers supports configuring IP failover through FRR. Review the [Configuring IP Failover on a Compute Instance](/docs/guides/ip-failover/) to learn more about IP Sharing / IP failover availability within each data center.
+{{</note>}}
+
+This guide covers using the open source [FRRouting (FRR)](http://docs.frrouting.org/en/latest/overview.html#about-frr) tool to configure IP failover with Linode Compute Instances. FRR is a routing service that uses BGP to monitor and fail over components in a high availability configuration. In a typical setup with IP failover, there is one **primary** Instance and one or more **secondary** Instances.
```

Comment:
should only be one primary and one secondary, if there are more than on failover it may cause issues and loops in certain areas of the network